src/test/java/jenkins/plugins/git/GitSCMSourceTraitsTest.java

```diff
@@ -188,6 +190,13 @@ public void pimpped_out() throws Exception {
                                         hasProperty("localBranch", is("**"))
                                 )
                         ),
+                        Matchers.<SCMSourceTrait>instanceOf(CleanAfterCheckoutTrait.class),
+                        Matchers.<SCMSourceTrait>instanceOf(CleanBeforeCheckoutTrait.class),
+                        Matchers.allOf(
+                                instanceOf(MessageExclusionTrait.class),
+                                hasProperty("extension",
+                                        hasProperty("excludedMessage", is("does not work")))
+                                )
```

Comment:
One more fix to priori merge error
```suggestion
                                )
                        ),
```